torchmdnet/priors/d2.py

```diff
@@ -0,0 +1,187 @@
+            [24.67, 1.639],  # 39 Y
+            [24.67, 1.639],  # 40 Zr
+            [24.67, 1.639],  # 41 Nb
+            [24.67, 1.639],  # 42 Mo
+            [24.67, 1.639],  # 43 Tc
+            [24.67, 1.639],  # 44 Ru
+            [24.67, 1.639],  # 45 Rh
+            [24.67, 1.639],  # 46 Pd
+            [24.67, 1.639],  # 47 Ag
+            [24.67, 1.639],  # 48 Cd
+            [37.32, 1.672],  # 49 In
+            [38.71, 1.804],  # 50 Sn
+            [38.44, 1.881],  # 51 Sb
+            [31.74, 1.892],  # 52 Te
+            [31.50, 1.892],  # 53 I
+            [29.99, 1.881],  # 54 Xe
+        ]
+    )
+    C_6_R_r[:, 1] *= 0.1  # Å --> nm
+
+    def __init__(
+        self,
+        cutoff,
+        max_num_neighbors,
+        atomic_numbers=None,
+        distance_scale=None,
+        energy_scale=None,
+        dataset=None,
+    ):
+        super().__init__()
+        self.cutoff = float(cutoff)
+        self.max_num_neighbors = int(max_num_neighbors)
+        self.atomic_numbers = list(
+            dataset.atomic_numbers if atomic_numbers is None else atomic_numbers
+        )
+        self.distance_scale = float(
+            dataset.distance_scale if distance_scale is None else distance_scale
+        )
+        self.energy_scale = float(
+            dataset.energy_scale if energy_scale is None else energy_scale
+        )
+
+        # Convert to interal units: nm and J/mol
+        # NOTE: float32 is overflowed, if m and J are used
+        self.distance_scale *= 1e9  # m --> nm
+        self.energy_scale *= 6.02214076e23  # J --> J/mol
+
+        # Distance calculator
+        self.distances = Distance(
+            cutoff_lower=0,
+            cutoff_upper=self.cutoff / self.distance_scale,
+            max_num_neighbors=self.max_num_neighbors,
+        )
+
+        # Parameters (default values from the reference)
+        self.register_buffer("Z_map", pt.tensor(self.atomic_numbers, dtype=pt.long))
+        self.register_buffer("C_6", self.C_6_R_r[:, 0])
+        self.register_buffer("R_r", self.C_6_R_r[:, 1])
+        self.d = 20
+        self.s_6 = 1
+
+    def reset_parameters(self):
+        pass
+
+    def get_init_args(self):
+        return {
+            "cutoff": self.cutoff,
+            "max_num_neighbors": self.max_num_neighbors,
+            "atomic_numbers": self.atomic_numbers,
+            "distance_scale": self.distance_scale,
+            "energy_scale": self.energy_scale,
+        }
+
+    def post_reduce(self, y, z, pos, batch):
+
+        # Get atom pairs and their distancence
+        ij, R_ij, _ = self.distances(pos * self.distance_scale, batch)
+
+        # No interactions
+        if ij.shape[1] == 0:
+            return y
+
+        # Compute the pair parameters
+        Z = self.Z_map[z[ij]]
+        C_6 = self.C_6[Z].prod(dim=0).sqrt()
+        R_r = self.R_r[Z].sum(dim=0)
+
+        # Compute pair contributions
+        f_dump = 1 / (1 + pt.exp(-self.d * (R_ij / R_r - 1)))
```

Comment:
I think you meant to call this `f_damp` rather than `f_dump`?  It's a damping function.